<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Login</string>
      </property>
      <widget class="QLabel" name="casa_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>761</width>
         <height>281</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="login_login_btn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>440</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="login_exit_btn">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>480</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>EXIT</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="password_checkBox">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>390</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="login_pass_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>131</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>350</y>
         <width>451</width>
         <height>69</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>User name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="login_username_editLine"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="login_password_editLine">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
